### DISCUSSION TITLE
How to ignore page?

### DISCUSSION THREAD

**Discussion by thatvirtualboy** [mmistakes/jekyll-theme-basically-basic#122] [Q&A] [answered]
How can I omit a page from showing up in the sidebar? I want to only have it linked to from a specific location (like a page containing a privacy policy, for example). Thanks!

**Comment by mmistakes** [comment, marked as answer]
https://github.com/mmistakes/jekyll-theme-basically-basic#navigation

**Comment by thatvirtualboy** [comment]
Kill the title - got it. Thanks!

**Comment by thatvirtualboy** [comment]
Shoot, so this isn't working. When I preview the site with `jekyll serve` it acts as expected. But when pushing live, the pages with the missing "titles" still show on my side bar. Can you confirm @mmistakes

**Comment by mmistakes** [comment]
Not sure, doesn't sound like a theme issue. If it works locally then you have it setup correctly there. You sure you're pushing those changes to your remote repo?

**Comment by mmistakes** [comment]
Also if you're hosting with GitHub Pages I believe they install some plugins that change the default behavior of titles. That would explain why it works for you locally but the behavior changes when pushing to GH Pages.

These are all the plugins they install. Pretty sure one of them auto-populates the `title` based off of the filename even if you leave it blank in the front matter.

https://pages.github.com/versions/

Could also be something with https://rubygems.org/gems/jekyll-titles-from-headings

Either way, it's not a theme issue.

**Comment by thatvirtualboy** [comment]
Hmm. Well thanks for looking into it. I don't see `jekyll-titles-from-headings` when I list out `gem list --local' but perhaps it wouldn't show there. Either way, would you have any suggestions on how to organize folders in such a way that I have a page that's only accessible via direct link? Thanks @mmistakes !

**Comment by thatvirtualboy** [comment]
Ah.. created a defined navigation in theme.yml, seems to work thus far!

**Comment by mmistakes** [comment]
Yeah, I was under the impression you created the defined navigation file from the documentation link I added a few comments before. That's the only way to get the menu how you want.
